src/System Application/App/Azure Key Vault/src/AzureKeyVaultImpl.Codeunit.al

```diff
@@ -30,18 +30,25 @@ codeunit 2202 "Azure Key Vault Impl."
         IsKeyVaultClientInitialized: Boolean;
         AzureKeyVaultTxt: Label 'Azure Key Vault', Locked = true;
         CertificateInfoTxt: Label 'Successfully constructed certificate from secret %1. Certificate thumbprint %2', Locked = true;
+        MissingSecretErr: Label 'The secret %1 is either missing or empty.', Comment = '%1 = Secret Name.';
 
     [NonDebuggable]
     procedure GetAzureKeyVaultSecret(SecretName: Text; var Secret: Text)
     begin
         // Gets the secret as a Text from the key vault, given a SecretName.
         Secret := GetSecretFromClient(SecretName);
+
+        if Secret.Trim() = '' then
+            Error(MissingSecretErr, SecretName);
```

Comment:
Feel free to won't fix and close the PR, just wondering if you want to fix the references that also does this trim check. Now they are no longer needed and we can clean those up. Ex. in AI module:
![image](https://github.com/user-attachments/assets/fc03eb08-9dab-4179-b7c6-5bb34827e513)

Reply:
I think we can keep this for now as well